doc/pygambit.user.rst

```diff
@@ -183,6 +183,9 @@ the top level index is the choice of the first player, the second level index of
 and so on.  Therefore, to create a two-player symmetric game, as in this example, the payoff matrix
 for the second player is transposed before passing to :py:meth:`.Game.from_arrays`.
 
+There is a reverse function :py:meth:`.Game.to_arrays` that produces
```

Comment:
Especially in the user guide, we try to make good use of the fact that we can inline Python code and the documentation generator will run the code and include the results.

It would be good then to illustrate how `to_arrays` works by re-generating the arrays from the previous example game.  Further it would be good to show how to use the `dtype` argument by an example.